forge-game/src/main/java/forge/game/trigger/TriggerBecomesSaddled.java

```diff
@@ -0,0 +1,45 @@
+package forge.game.trigger;
+
+import java.util.Map;
+
+import forge.game.ability.AbilityKey;
+import forge.game.card.Card;
+import forge.game.spellability.SpellAbility;
+import forge.util.Localizer;
+
+public class TriggerBecomesSaddled extends Trigger {
+
+    public TriggerBecomesSaddled(final Map<String, String> params, final Card host, final boolean intrinsic) {
+        super(params, host, intrinsic);
+    }
+
+    @Override
+    public boolean performTest(Map<AbilityKey, Object> runParams) {
+        if (!matchesValidParam("ValidSaddled", runParams.get(AbilityKey.Card))) {
+            return false;
+        }
+        if (hasParam("FirstTimeSaddled")) {
+            Card v = (Card) runParams.get(AbilityKey.Card);
+            if (v.getTimesSaddledThisTurn() != 1) {
```

Comment:
@tool4ever we had some other problems with such First Time triggers, right? Does this work that way?

Reply:
well most likely saddling won't start happening simultaneous
so we should be relatively safe